frontend/src/shared/pill/pill.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="lf-pill" :class="props.color + ' ' + props.type">
+    <slot />
+  </div>
+</template>
```

Comment:
For all re sharable components, Gasper is adding them under ui-kit.
Can you check if there's already any compoennt there that fits this need?
You can run storybook locally (check package.json) to check the available components.
If it doesn't exist, please create this component in the ui-kit folder with the same structure as the remaining components

Reply:
This doesn't exist there. I moved this component under the ui-kit folder

Reply:
I pushed this fix